src/sql/base/browser/ui/table/table.ts

```diff
@@ -135,8 +144,50 @@ export class Table<T extends Slick.SlickData> extends Widget implements IDisposa
 		this.mapMouseEvent(this._grid.onDblClick, this._onDoubleClick);
 		this._grid.onColumnsResized.subscribe(() => this._onColumnResize.fire());
 
-		this._grid.onKeyDown.subscribe((e, args: Slick.OnKeyDownEventArgs<T>) => {
+		this._grid.onKeyDown.subscribe(async (e, args: Slick.OnKeyDownEventArgs<T>) => {
 			const evt = (e as JQuery.TriggeredEvent).originalEvent as KeyboardEvent;
+
+			/**
+			 * Adding a keyboard shortcut Alt+left/right arrow to change column width. This is a
+			 * workaround to provide an alternative to column header mouse drag based resizing.
+			 * TODO: Make the shortcut configurable or provide a way to focus on column headers
+			 * so that users can focus a header and then use the arrow keys to resize columns.
+			 */
+			const stdEvt = new StandardKeyboardEvent(evt);
+
+			if (stdEvt.altKey && stdEvt.shiftKey && stdEvt.keyCode === KeyCode.KeyS) {
```

Comment:
I think there is a resizable option for each column, make sure you respect that option.